src/main/java/com/github/wovnio/wovnjava/FilterServletOutputStream.java

```diff
@@ -36,4 +37,14 @@ public void close() throws java.io.IOException {
     public void flush() throws IOException {
         stream.flush();
     }
+
+    @Override
+    public boolean isReady() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public void setWriteListener(WriteListener writeListener) {
+        throw new UnsupportedOperationException();
+    }
```

Comment:
i think you would just return true for isReady, and do nothing in `setWriteListener`